Sources/MapboxNavigation/Feedback/FeedbackViewController.swift

```diff
@@ -150,6 +145,7 @@ public class FeedbackViewController: UIViewController, DismissDraggable, UIGestu
     
     required public init?(coder aDecoder: NSCoder) {
         eventsManager = aDecoder.decodeObject(of: [NavigationEventsManager.self], forKey: "NavigationEventsManager") as? NavigationEventsManager
+        self.type = FeedbackViewControllerType.activeNavigation
```

Comment:
Is it okay to have a hardcoded type here?

Reply:
It's like a default type, if a user want to use another type, he need to use another init.